.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Description
+
+Please explain the changes you made here.
+Explain the **motivation** for making this change. What existing problem does the pull request solve?
+
+### Checklist
+
+- [ ] Code compiles correctly
+- [ ] All tests passing
+- [ ] Extended the README / documentation, if necessary
+- [ ] code formatting
```

Comment:
code formatting is important but hard for contributors since we don't have a defined style guide.